sphinx/texinputs/sphinx.sty

```diff
@@ -397,6 +402,15 @@
 %          macro prefix   option prefix   legacy option    init value
 \spx@tempa{spx@pre@}      {pre_}          {verbatimborder} {0.4pt}
 \spx@tempa{spx@topic@}    {div.topic_}    {shadowrule}     {0.5pt}% mod. at 7.4.0
+\spx@tempa{spx@contents@} {div.contents_} {shadowrule}     {0.5pt}% 8.1.0
+\spx@tempa{spx@sidebar@}  {div.sidebar_}  {shadowrule}     {1pt}%   8.1.0
+% let legacy shadowrule key set all topic/contents/sidebar border
+% keys to the common value given by user to shadowrule
+\def\KV@sphinx@shadowrule #1{%
+    \@nameuse{KV@sphinx@div.topic_border-width}{#1}%
+    \@nameuse{KV@sphinx@div.contents_border-width}{#1}%
+    \@nameuse{KV@sphinx@div.sidebar_border-width}{#1}%
```

Comment:
This is really me nitpicking here, but I think we use a 2-space indent in general instead of a 4-space.